Classes/Integration/NetlogixIntegration.php

```diff
@@ -159,6 +160,51 @@ private static function isInApp(string $path): bool
         return true;
     }
 
+    private static function scrubVariablesFromFrame($traceFunction, $frameVariables): array
```

Comment:
```suggestion
    private static function scrubVariablesFromFrame(string $traceFunction, array $frameVariables): array
```
Otherwise `$frameVariables` may not be an array which would break the return in line 166